@nav-frontend/popover-styles/src/index.css

```diff
@@ -7,7 +7,7 @@
   z-index: var(--popover-z-index);
   background-color: var(--navds-color-background);
   box-shadow: var(--popover-shadow);
-  border: var(--navds-border);
+  border: var(--navds-border-on-light);
```

Comment:
Hva er tanken med `-on-light`? Må vi da også endre kode i andre komponenter?

Reply:
Tok inspirasjon fra https://designsystem.morningstar.com/visual-language/constants/?version=3.0.5&tab=current#borders
Har allerede endret bruken til `on-light` i andre komponenter

Løser problemet med at man ikke kan sette tokens slik:
```
border: { value : "" }
border: { radius: { "" } }
```
eller slikt
```
border: { value: "", radius: { value: "" } }
```

Reply:
Ah, ser ut som de bruker det for å støtte dark-mode e.l. For meg virker det rart å ha en `-on-light` uten en `-on-dark`, men vi har kanskje planer om det på sikt? Hva med noe mer generisk som `-default` istedenfor?

Reply:
Med på den, endrer til default.